### ISSUE TITLE
Add DeepAlpha - Time series ML for crypto price prediction

### ISSUE THREAD

**Issue by stefanoviana** [Alro10/deep-learning-time-series#28]
Suggesting [DeepAlpha](https://github.com/stefanoviana/deepalpha).

Time series ML framework for crypto price prediction using LightGBM + XGBoost + TFT (Temporal Fusion Transformer) with 72 features. 70.9% walk-forward accuracy. MIT licensed.
